tests/smoke-npm-group-rules.yaml

```diff
@@ -0,0 +1,38 @@
+input:
```

Comment:
I think you need to run `dependabot update -f tests/smoke-npm-group-rules.yaml -o tests/smoke-npm-group-rules.yaml` to append the generated `output:` object to this file so we can cache the requests and make sure it has the same results.